<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathExpBlockEditorWidget</class>
 <widget class="QWidget" name="MathExpBlockEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>vars list (comma-separated)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="varsEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="toolbar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>math expression edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="testExprBtn">
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="exprEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
